lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/launch/JavaLaunchCommandBuilder.java

```diff
@@ -168,6 +180,28 @@ public JavaLaunchCommandBuilder addOption(final String option) {
     return addOption(JVMOption.parse(option));
   }
 
+  /**
+   * Pass values of the properties specified in the propNames array as <code>-D...</code>
+   * command line parameters. Currently used only to pass logging configuration to child JVMs processes.
+   *
+   * @param vargs     List of command line parameters to append to.
+   * @param copyNull  create an empty parameter if the property is missing in current process.
+   * @param propNames property names.
+   */
+  private static void propagateProperties(
```

Comment:
It is only used inside the class even it doesn't access any instance variables. Agree it is just a preference. This one is more like a utility method.  I am fine with it.